ui/app/AppLayouts/Wallet/popups/swap/SwapModalAdaptor.qml

```diff
@@ -203,7 +201,7 @@ QObject {
             }
             readonly property LeftJoinModel joinModel: LeftJoinModel {
                 leftModel: submodel
-                rightModel: root.filteredFlatNetworksModel
+                rightModel: root.swapStore.flatNetworks
```

Comment:
why not use the root.filteredFlatNetworksModel instead which filters testnet/mainnet networks already?

Reply:
It doesn't work correctly that way, the `LeftJoinModel` is not filtering so you'll get ghost/incorrect entries that should not be there; I'll fix it as part of the followup PR here: https://github.com/status-im/status-desktop/issues/14783